src/elements/OcBreadcrumb.vue

```diff
@@ -25,7 +25,7 @@ export default {
   release: "1.0.0",
   props: {
     items: {
-      type: Array,
+      type: [Boolean, Array],
```

Comment:
Is that reasonable? What Happens If Set to true? What when Set to false

Reply:
Problem was, that `activeRoute` returns `false` in the home directory. Since `home` is now part of the breadcrumb, a type warning pops up.

Reply:
but the correct fix is in phoenix to return [] instead of false

Reply:
I'm wondering if root should be the first entry in the array so we can navigate to it properly.
Currently, we are pulling off this funky stunt to navigate back to the root: https://github.com/owncloud/phoenix/blob/master/apps/files/src/components/FilesTopBar.vue#L6

Reply:
```json
[
    {"index": 0, "text": "",       "to": "/files/list"}, //Suggestion
    {"index": 1, "text": "Funky",  "to": "/files/list/Funky"},
    {"index": 2, "text": "wefwef", "to": "/files/list/Funky/Demo"}
]
```

Reply:
sounds reasonable - but unrelated to this PR -> move to own PR